lib/db_connection.rb

```diff
@@ -0,0 +1,16 @@
+require 'pg'
+
```

Comment:
Good use of a class here. Keeps your methods clean in the other two classes, and avoids repeating yourself